### DISCUSSION TITLE
How to remove excessive padding while adding a telemetry (label + value) ?

### DISCUSSION THREAD

**Discussion by bangpradyumna** [nasa/openmct#7563] [Community Q&A] [answered]
Hello,
I've been creating a display in openmct using telemetry from YAMCS using openmct-yamcs plugin. However, Whenever I add a telemetry such as a Switch status (boolean) or any other telemetry (string/int/float) - there is always some excessive padding that is added by openmct itself. 

How do I reduce this or remove it altogether ? Attaching a picture for reference - in this screenshot, the label doesn't get enough space but the value takes excessive space. This leads to alot of wastage of space, any way to customize it ?

![image](https://github.com/nasa/openmct/assets/6296142/01bd40c9-e650-4444-9e03-81382496ab98)

**Comment by ozyx** [comment, marked as answer]
Hi @bangpradyumna ,

I'm going to create an issue from this. There is currently no way to customize this other than altering CSS. I think I've found an old, open issue that [references the padding here as well](https://github.com/nasa/openmct/issues/2661).

Closing to track in [this issue](https://github.com/nasa/openmct/issues/7564)
